### PR TITLE
Fixes doc_749

### PR DESCRIPTION
**Pull request by TobyHFerguson**
Adds AWS cli commands showing how to get the outputs

### CODE REVIEW

**Comment by kaitlynmichael** [comment]
@TobyHFerguson it looks like you need to sign the CLA for us to merge these changes.

**Comment by TobyHFerguson** [comment]
@kaitlynmichael Hmm - I'm not sure what to do. I signed the CLA some time ago (and again today!). All of my email addresses that I know about are associated with this github profile (the one with the funny green avatar, linked to the TobyHFerguson account). I can't find the exact email address that was used for this commit, but as I use this one user all the time I assume its one of those addresses. 

Do you have any ideas on how to solve this?

**Comment by TobyHFerguson** [comment]
Figured it out - my old gmail address was missing. CLA now signed :-)

File: content/rc/how-to/creating-iam-resources/cloudformation.md [hunk 1]
```diff
@@ -12,15 +12,48 @@ The following link uses CloudFormation to create a stack using the AWS console:
...
 ```
+export AWS_PROFILE=YOUR_PROFILE_HERE
 aws cloudformation create-stack --stack-name RedisCloud --template-url \
 https://s3.amazonaws.com/iam-resource-automation-do-not-delete/RedisCloud.yaml \
---capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_NAMED_IAM CAPABILITY_IAM \
---profile YOUR_PROFILE_HERE
+--capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_NAMED_IAM CAPABILITY_IAM
 ```
 
+You can track the status of the cloud formation thus:
```
Comment (kaitlynmichael) [inline content/rc/how-to/creating-iam-resources/cloudformation.md:26]:
```suggestion
You can track the status of the cloud formation with the following command:
```

File: content/rc/how-to/creating-iam-resources/cloudformation.md [hunk 2]
```diff
@@ -12,15 +12,48 @@ The following link uses CloudFormation to create a stack using the AWS console:
...
+```
+The data needed to complete the creation of a Cloud Account is shown as `Output Key` and `Output Value` pairs.
+
+For the two secrets (`accessSecretKey` and `consolePassword`) you'll need to use the AWS secretmanager CLI - the value you'll need has a key of `SecretString`:
+
+```
+aws secretsmanager get-secret-value --secret-id=/redislabsuser/secret_access_key
+```
+
+We recommend for the `consolePassword` you use yaml output - it makes decoding the required value easier
```
Comment (kaitlynmichael) [inline content/rc/how-to/creating-iam-resources/cloudformation.md:38]:
```suggestion
We recommend using yaml output for the `consolePassword`, as it makes decoding the required value easier.
```

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**